test/transpile.test.js

```diff
@@ -14,35 +15,85 @@ function cleanCRLF(str) {
 }
 
 var transpileDir = path.join(__dirname, 'transpile');
-var expectedFilePath = path.join(transpileDir, 'expected.js');
-var expected = readCleanCRLF(expectedFilePath);
-
-/**
- * Execute the binary and capture the stdout
- */
-const { exec } = require('child_process');
-const child = exec('node ./bin/dogescript.js ./test/transpile/source.djs',
-  (err, stdout, stderr) => {
-    if(err)
-    {
-      console.error(`exec error: ${err}`);
-      return;
-    }
+
+// Pop a directory off it's path (os safe)
+function dotdotSlash(filePath) {
```

Comment:
What's the difference between this and `getFolderName` below?  Why not use Node's path module for this?